src/Common/Doctrine/ValueObject/AbstractFile.php

```diff
@@ -201,9 +201,9 @@ public function __toString(): string
         return (string) $this->fileName;
     }
 
-    public static function fromString(string $fileName): self
+    public static function fromString(string $fileName): ?self
```

Comment:
Yeah thanks! I'm using master in my project and that doesn't have the php7 typehints yet so that one went unnoticed